src/iso19111/io.cpp

```diff
@@ -6070,7 +6070,7 @@ static bool isGeocentricStep(const std::string &name) {
 // ---------------------------------------------------------------------------
 
 static bool isProjectedStep(const std::string &name) {
-    if (name == "etmerc" || name == "utm" ||
+    if (name == "stmerc" || name == "utm" ||
```

Comment:
name == "etmerc" should probably be kept as well. projectionMethodMappings[] in include/proj/internal/coordinateoperation_constants.hpp has only mapping for "tmerc"

Reply:
Around line 6819
```
} else if (step.name == "etmerc") {
        mapping = getMapping(EPSG_CODE_METHOD_TRANSVERSE_MERCATOR);
```
should also test || step.name == "stmerc"

Reply:
And around line 7071
```
        if (step.name == "etmerc") {
            methodMap.set("proj_method", "etmerc");
        }
```
there should be a similar case for stmerc